build/xtask/src/dist.rs

```diff
@@ -1656,6 +1933,7 @@ fn build(
             "-C link-arg=-z -C link-arg=common-page-size=0x20 \
              -C link-arg=-z -C link-arg=max-page-size=0x20 \
              -C llvm-args=--enable-machine-outliner=never \
+             -Z emit-stack-sizes \
```

Comment:
do you know if there is any work tracking this not being nightly only? We still have a goal (dream?) of being able to build hubris with a stable toolchain.

Reply:
The path to stabilization appears rocky, unfortunately:
https://github.com/rust-lang/rust/issues/54192